pysidt/sidt.py

```diff
@@ -339,6 +339,23 @@ def read_nodes(file):
 
 
 class MultiEvalSubgraphIsomorphicDecisionTree(SubgraphIsomorphicDecisionTree):
+    """
+    Makes prediction for a molecule based on multiple evaluations.
+
+    Args:
+        `decomposition`: method to decompose a molecule into groups.
```

Comment:
Broadly this is true, we are decomposing them into group contributions but on a technical standpoint this doesn't make RMG Group objects so maybe "groups" => "substructure contributions"